README.md

```diff
@@ -57,14 +56,19 @@ To try out the ForgeRock JavaScript SDK, perform these steps:
    See [Enabling CORS Support](https://sdks.forgerock.com/js/01_prepare-am/#enabling-cors-support) in the Documentation.
 2. Create an authentication tree in AM.
 
-   See [Creating a User Authentication Tree](https://sdks.forgerock.com/js/01_prepare-am/#creating-a-user
- -authentication-tree) in the Documentation.
-4. Clone this repo:
+   See [Creating a User Authentication Tree](https://sdks.forgerock.com/js/01_prepare-am/#creating-a-user-authentication-tree) in the Documentation.
+3. Clone this repo:
 
     ```
     git clone https://github.com/ForgeRock/forgerock-javascript-sdk.git
     ```
 
+4. Build the ForgeRock JavaScript SDK with NPM:
```

Comment:
Before one can build the project, they have to install the dependencies `npm install`.